pycloudlib/openstack/instance.py

```diff
@@ -91,7 +91,7 @@ def console_log(self):
         while time.time() < start + 180:
             response = self.conn.compute.get_server_console_output(self.server)
             if response:
-                return response
+                return response['output']
```

Comment:
Looks good had to validate what python openstack python client returned (as their get_and_munchify() function in /usr/lib/python3/dist-packages/openstack/cloud/meta.py led me to believe that the top-level "output" key wasn't going to be present)
```bash
(Pdb) inst.console_log.keys()
dict_keys(['output'])
```